new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1189,6 +1189,28 @@ public static void checkPodDoesNotExist(String podName, String domainUid, String
                 podName, namespace)));
   }
 
+
+  /**
+   * Check pod does not exist in the specified namespace.
+   *
+   * @param podName pod name to check
+   * @param namespace the namespace in which to check whether the pod exists
+   */
+  public static void checkPodDoesNotExist(String podName, String namespace) {
```

Comment:
you can just call checkPodDoesNotExist(podName, null, namespace)

Reply:
done